src/config.rs

```diff
@@ -10,7 +10,9 @@ pub trait ConfigBuilderExt {
     /// rustls-native-certs
     #[cfg(feature = "rustls-native-certs")]
     #[cfg_attr(docsrs, doc(cfg(feature = "rustls-native-certs")))]
-    fn with_native_roots(self) -> ConfigBuilder<ClientConfig, WantsTransparencyPolicyOrClientCert>;
+    fn with_native_roots(
```

Comment:
I think the rustdoc comment for this trait fn should mention why it might return `None`, and to suggest using `with_webpki_roots` in that circumstance.